src/components/LogoutPage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
```

Comment:
I don't think there's need for a logout page. Once a user logs out, they should be redirected to where they can either login or signup.